app/src/main/res/layout/activity_timeline.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".TimelineActivity">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/rvTweets"
```

Comment:
The convention is that xml files use snake_case, whereas Java/Kotlin files use camelCase.
So this would be `rv_tweets` for example

Reply:
No need to fix urgently